.travis.yml

```diff
@@ -1,16 +1,26 @@
+# Use the default infrastructure (Container-based Ubuntu 14.04)
 sudo: false
 
-language: node_js
+language: python
 
-node_js: node
+python:
+  - 3.5
+
+addons:
+  apt:
+    packages:
+      - gettext
+      - nodejs
 
 cache:
   directories:
   - node_modules
 
 install:
   - npm install
```

Comment:
One call: `pip install requirements`

Reply:
Are you saying the `npm install` should be moved somewhere else?

Reply:
It’s already done in the requirements target.

Reply:
When I called `make requirements` in a prior version of this PR, it didn't seem to run `npm install` (I got an error complaining that it was missing an executable)?

Reply:
Perhaps you hadn't installed Node.js. Try again.